### PR TITLE
add DEMISTO_SDK_CONTENT_PATH

### PR DESCRIPTION
**Pull request by ilaner**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-hq.paloaltonetworks.local/browse/CIAC-3636

## Description
Adding the content path env variable to handle executing the demisto-sdk not from the content path.

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation
- [ ] [VS Code Extension](https://docs.gitlab.com/ee/ci/yaml/#tags)
  - [ ] Functionality implemented in the extension - <u>\<link-to-pr-here\></u>
  - [ ] N/A - Functionality cannot be implemented in the extension because <u>\<add-reason-here\></u>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,11 @@ The library uses python 3.8+.
 
 ---
 
+### Content path
+
+The **demisto-sdk** is build to work under [content](https://github.com/demisto/content) repo or a fork of it.
+It should be executed in a directory of a subdirectory of **content**. However, it is possible to set the `DEMISTO_SDK_CONTENT_PATH` environment variable to the **content** path.
```
Comment (dorschw) [inline README.md:51]:
```suggestion
The **demisto-sdk** is made to work with Cortex content, structured similar to the [official Cortex content repo](https://github.com/demisto/content).

Demisto-SDK commands work best when called from the content directory or any of its subfolders.
To run Demisto-SDK commands from other folders, you may set the `DEMISTO_SDK_CONTENT_PATH` environment variable.
```

File: demisto_sdk/commands/common/content/content.py [hunk 2]
```diff
@@ -68,8 +72,12 @@ def git() -> Repo | None:
             1. Should be called when cwd inside content repository.
         """
         try:
-            repo = Repo(Path.cwd(), search_parent_directories=True)
+            if content_path := os.getenv('DEMISTO_SDK_CONTENT_PATH'):
+                repo = Repo(content_path)
+            else:
+                repo = Repo(Path.cwd(), search_parent_directories=True)
         except InvalidGitRepositoryError:
+            logger.debug('Git repo was not found.')
```
Comment (dorschw) [inline demisto_sdk/commands/common/content/content.py:80]: 1. isn't it an error/warning?
2. does it replace the message we already print?

**Review by dorschw — COMMENTED** [review]
Nice

File: demisto_sdk/commands/common/content/content.py (hunk 2 above)
Comment (ilaner) [reply demisto_sdk/commands/common/content/content.py:81]: This is just a debug message.
The logic stays the same (returning `None` and let the command decide what to do with it)

**Review by dorschw — APPROVED** [review]

File: demisto_sdk/commands/common/tools.py [hunk 3]
```diff
@@ -1583,8 +1583,14 @@ def get_content_path() -> str:
     Returns:
         str: Absolute content path
     """
+    # avoid circular import
```
Comment (dorschw) [inline demisto_sdk/commands/common/tools.py:1586]:
```suggestion
```

**Pull request merged**